moped-editor/src/queries/project.js

```diff
@@ -96,6 +96,7 @@ export const SUMMARY_QUERY = gql`
       entity_id
       moped_entity {
         entity_name
+        entity_id
```

Comment:
the cache actually breaks if you define a `typePolicy` don't include the PK property in every instance of the type.